transcoder.py

```diff
@@ -506,7 +506,7 @@ def updateCoverMp3(lossyFileName, artworkFileName):
     import tempfile
     from shutil import copyfile  # Use copyfile b/c this will *not* copy rights (which is error prone on gvfs/samba)
 
-    log('- embedding album art to ' + lossyFileName) 
+    log('  - embedding album art to ' + lossyFileName) 
```

Comment:
Extra spatie?

Reply:
Ja, dat maakt het bij de uitvoer overzichtelijker

Reply:
Doen we eigenlijk nergens. En we gaan sowieso de bezem hier doorheen halen en een andere todo. Dus graag weghalen

Reply:
Oke ik zal ze allemaal weghalen dan